lib/vagrant-service-manager/adb_executor.rb

```diff
@@ -0,0 +1,40 @@
+module VagrantPlugins
+  module ServiceManager
+    VERSION_CMD = {
+      docker: "docker version --format '{{.Server.Version}}'",
+      openshift: "oc version | grep 'oc' | grep -oE '[0-9.]+'"
+    }
+
+    class ADBExecutor
+      LABEL = 'servicemanager.commands.install_cli.message'
+
+      def initialize(binary_type, machine, ui, options)
+        @type = binary_type
+        @machine = machine
+        @ui = ui
+        @options = options
+      end
+
+      def execute
+        binary_exists = true
+        version = @options['--cli-version']
+        path = @options['--path']
+
+        version = PluginUtil.execute_once(@machine, @ui, VERSION_CMD[@type.to_sym]) if version.nil?
```

Comment:
`PluginUtil` is a helper class right? By including it you can call these helper methods directly without always having to qualify `PluginUtil`. This is just a readability thing. IMO the code becomes more concise. Is that not the Ruby way?